src/addons/Confirm/Confirm.js

```diff
@@ -46,12 +46,16 @@ class Confirm extends Component {
 
     /** Whether or not the modal is visible. */
     open: PropTypes.bool,
+
+    /** A modal can vary in size */
```

Comment:
This should say `Confirm` and have a period:
```diff
-    /** A modal can vary in size */
+    /** A Confirm can vary in size. */
```

Reply:
Note, I tried to update this but it is on @ashurbeyli's `master` branch which I cannot push to.